src/index.html

```diff
@@ -11,6 +11,18 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    >
+      <input class="one" data-qa="keypress"
+      placeholder="Try &quot;Los Angeles&quot;">
+    </form>
```

Comment:
apply code styling recommendations
```suggestion
    <form 
      data-qa="big"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
    >
      <input 
        class="one" 
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```